src/Slider.jsx

```diff
@@ -532,7 +532,7 @@ class Rheostat extends React.Component {
     }
     const horizontalPercentage = ((x - sliderBox.left) / sliderBox.width) * PERCENT_FULL;
     if (direction === DIRECTIONS.RTL) {
-      return 100 - horizontalPercentage;
+      return horizontalPercentage;
```

Comment:
I don't think this change is correct. If so the `horizontalPercentage` is the same in RTL and LTR. [The bigger issue](https://github.com/airbnb/rheostat/issues/282#issuecomment-1380786644) is that `react-with-direction` is not auto-flipping the left/right CSS.